addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1336,18 +1399,16 @@ var SnippetEditor = dragAndDropCommonMenuEditor.extend({
 
                 // Reload the images.
                 gridUtils._reloadLazyImages(this.$target[0]);
-            } else {
-                // If the column comes from a snippet that doesn't toggle the
-                // grid mode on drag, store its width and height to use them
-                // when the column goes over a grid dropzone.
-                this._storeItemWidthAndHeight();
             }
             // Storing the starting top position of the column.
             this.dragState.columnTop = this.$target[0].getBoundingClientRect().top;
             this.dragState.isColumn = true;
             // Deactivate the snippet so the overlay doesn't show.
             this.trigger_up('deactivate_snippet', {$snippet: self.$target});
         }
+        // Store the dragged element width and height to use them when the
```

Comment:
Should come here as the snippet can start from a non grid zone to go in a grid zone.